src/main/groovy/wooga/gradle/build/unity/ios/IOSBuildPlugin.groovy

```diff
@@ -388,4 +394,43 @@ class IOSBuildPlugin implements Plugin<Project> {
         archiveDSYM.configure({ it.mustRunAfter(xcodeExport) })
         tasks.named(BasePlugin.ASSEMBLE_TASK_NAME).configure({ it.dependsOn(xcodeExport, archiveDSYM, collectOutputs) })
     }
+
+    /***
+     * Applies the asdf plugin, which manages the installation and usage of the asdf tool which manages
+     * multiple language runtime versions on a per-project basis.
+     * (https://github.com/asdf-vm/asdf)
+     * @param project
+     */
+    void applyAsdfPlugin(Project project) {
+
+        /**
+             asdf {
+                version "0.10.0"
+             }
+
+             asdfRuby {
+                 gem "cocoapods"
+                 gem "cocoapods-art"
+                 gem "cocoapods-pod-linkage"
+             }
+         */
+
+        project.pluginManager.apply(AsdfPlugin.class)
+        project.pluginManager.apply(RubyPlugin.class)
+
+        def asdf = project.extensions.getByType(AsdfPluginExtension)
+        asdf.version.convention(IOSBuildPluginConventions.asdfVersion.getStringValueProvider(project))
+
+        def ruby = project.extensions.getByType(RubyPluginExtension)
+        ruby.gem("cocoapods")
+        ruby.gem("cocoapods-art")
+        ruby.gem("cocoapods-pod-linkage")
+
+        // This was set because...?
+        def installRubyTask = project.tasks.named("asdfInstallRuby ")
```

Comment:
There is an extra space at the end of `asdfInstallRuby`, I guess this may cause issues.

Reply:
It did not work either. I will remove it for now since I dd not want to add a hook after project has been configured.